revscoring/datasources/types.py

```diff
@@ -66,3 +66,9 @@ def __init__(self, id=None, name=None, editcount=None, registration=None,
             if block_reason is not None else None
         self.block_expiry = str(block_expiry) \
             if block_expiry is not None else None
+
+        if globalinfo is None:
```

Comment:
We should probably store globalinfo.  That way, other features can make use of it.  We really shouldn't be figuring out "globalage" within the datasource itself.  I think the datasource should just provide the raw data and features can figure out how they'll make use of that data.